header/actor-zeta/base/actor_address.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
```

Comment:
Why do I need a file?

Reply:
idk, I got it from merging with mainline. I can delete it if it doesn't break something

Reply:
make the file look like in the developer branch.

Reply:
pulled it from develop